home/dot/gitconfig

```diff
@@ -1,19 +1,39 @@
 [user]
-	email = gary.verhaegen@gmail.com
-	name = Gary Verhaegen
+    email = gary.verhaegen@gmail.com
+    name = Gary Verhaegen
```

Comment:
This is a tab -> 4 spaces replacement. Note that these changes happened over about 10 years so I don't remember how or when they happened; comments and questions are best focused on understanding current state.